.claude/skills/baseline-reindex/SKILL.md

```
@@ -0,0 +1,102 @@
+OPCT_ENABLE_ADM_BASELINE="1" opct adm baseline indexer --log-level=debug
+
+# List baseline results
+opct adm baseline list
+
+# Publish a result
+OPCT_ENABLE_ADM_BASELINE="1" opct adm baseline publish --log-level=debug <artifact.tar.gz>
+```
+
+## S3 bucket structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)